tests/assets/input/keyboard.html

```diff
@@ -10,13 +10,13 @@
       let textarea = document.querySelector('textarea');
       textarea.focus();
       textarea.addEventListener('keydown', event => {
-        log('Keydown:', event.key, event.code, event.which, modifiers(event));
+        log('Keydown:', event.key, event.code, event.location, modifiers(event));
```

Comment:
Let's stringify location as in https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/location